NetworkingSampleApp/NetworkingSampleApp/Scene/SampleViewScene/SampleViewModel.swift

```diff
@@ -23,7 +23,7 @@ final class SampleViewModel {
         #endif
         
         return APIManager(
-            urlSession: URLSession.shared,
+            responseProvider: URLSession.shared,
```

Comment:
As mentioned in `APIManager` I would rather see the `urlSession` parameter name here